packages/backend/src/miro/miro.service.ts

```
@@ -0,0 +1,380 @@
+  }
+
+  /**
+   * One authenticated GET. Miro's status codes are translated into Nest
+   * exceptions with messages that describe the problem WITHOUT echoing the
+   * token.
+   */
+  private async getJson<T>(url: string, token: string): Promise<T> {
+    let res: Response;
+    try {
+      res = await fetch(url, {
+        headers: { Authorization: `Bearer ${token}`, Accept: 'application/json' },
```

ESLint

[error] Replace `·Authorization:·`Bearer·${token}`,·Accept:·'application/json'` with `⏎··········Authorization:·`Bearer·${token}`,⏎··········Accept:·'application/json',⏎·······`

(prettier/prettier)